### PR TITLE
Update for v2.11.1

### PR DESCRIPTION
**Pull request by 1azunna**
- Updated API for v2.11.1

### CODE REVIEW

**Comment by psiinon** [comment]
You will need to sign off the DCO - see https://github.com/zaproxy/zap-api-go/pull/13/checks?check_run_id=5129506312
BTW this repo is still configured to use Travis, which probably wont work :( We need to change that...

**Comment by 1azunna** [comment]
> You will need to sign off the DCO - see https://github.com/zaproxy/zap-api-go/pull/13/checks?check_run_id=5129506312 BTW this repo is still configured to use Travis, which probably wont work :( We need to change that...

Sure. Will update to use github actions

File: README.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+To regenerate the API code you will need the repos [zaproxy](https://github.com/zaproxy/zaproxy) and [zap-extensions](https://github.com/zaproxy/zap-extensions) checked out at the same level as this one.
+
+You should typically generate the core API calls from the latest release tag e.g.:
+
+```
+cd zaproxy
+git fetch upstream -t
+git checkout tags/v2.11.1
+git checkout tags/v2.11.1 -b goapi/2-11-1
```
Comment (psiinon) [inline README.md]: This line isnt needed - I removed it in my PR as this was flagged in a review ;)

**Review by psiinon — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (1azunna) [reply README.md]: Updated

**Comment by thc202** [comment]
Could you do a separate PR for the use of GitHub Actions?

**Comment by 1azunna** [comment]
@thc202 I have created the PR for the github actions. #14

**Comment by 1azunna** [comment]
Cleaned up this PR @psiinon @thc202

**Comment by thc202** [comment]
Could you rebase to pick #14?

**Comment by 1azunna** [comment]
done @thc202

**Review by kingthorin — APPROVED** [review]

**Comment by thc202** [comment]
Will check asap.

File: zap/network_generated.go [hunk 2]
```diff
@@ -0,0 +1,196 @@
+// Zed Attack Proxy (ZAP) and its related class files.
```
Comment (thc202) [inline zap/network_generated.go]: Better remove this one, most of the endpoints will not work with 2.11.

Comment (kingthorin) [reply zap/network_generated.go]: Sorry I should have caught that, excellent point.

Comment (1azunna) [reply zap/network_generated.go]: This exists on the python API as well.

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Pull request merged**